.pre-commit-config.yaml

```diff
@@ -72,7 +72,7 @@ repos:
     hooks:
       - id: isort
         name: isort (cython)
-        args: ["--config-root=python/", "--resolve-all-configs"]
+        args: ["--resolve-all-configs"]
```

Comment:
Without this isort will look for a configuration file under `python/`, and we want it to use `pyproject.toml` in the root of the repository.